src/pages/Staking/AuthorizeStakingApps/AuthorizeApplicationsCardCheckbox/index.tsx

```diff
@@ -177,40 +270,122 @@ export const AuthorizeApplicationsCardCheckbox: FC<
+                    ? minAuthAmount
+                    : WeiPerEther.toString()
+                  : "0"
+              }
+              helperText={`Minimum ${formatTokenAmount(minAuthAmount)} T for ${
+                appAuthData.label
+              }`}
+            />
+          </GridItem>
+        )}
       </Grid>
+      {hasPendingDeauthorization && (
+        <>
+          <BodyMd>Pending Deauthorization</BodyMd>
+          <InfoBox
+            p="6"
+            direction={{ base: "column", sm: "row" }}
+            justifyContent="space-between"
+            alignItems="center"
+          >
+            <H5>
+              {formatTokenAmount(pendingAuthorizationDecrease)}{" "}
+              <BodyLg as="span">T</BodyLg>
+            </H5>
+            <Box minWidth="220px">
+              <>
+                <Progress
+                  h="2"
+                  borderRadius="md"
+                  colorScheme="brand"
+                  value={
+                    remainingAuthorizationDecreaseDelay === "0"
+                      ? 100
+                      : !isDeauthorizationReqestActive
+                      ? 0
+                      : calculatePercenteage(
+                          remainingAuthorizationDecreaseDelay,
+                          stakingAppAuthDecreaseDelay
+                        )
+                  }
+                />
+                <BodySm>
+                  {!isDeauthorizationReqestActive &&
+                    "Deauthroziation request not activated"}
+                  {isDeauthorizationReqestActive &&
+                    remainingAuthorizationDecreaseDelay === "0" &&
+                    "Completed"}
+                  {isDeauthorizationReqestActive &&
+                    remainingAuthorizationDecreaseDelay !== "0" &&
+                    deauthorizationCreatedAt !== undefined && (
+                      <>
+                        Available:{" "}
+                        <BodySm as="span" color="brand.500">
+                          {formatDate(
+                            +deauthorizationCreatedAt +
+                              +stakingAppAuthDecreaseDelay
+                          )}
+                        </BodySm>
+                      </>
+                    )}
+                </BodySm>
+              </>
+            </Box>
+            <Button
+              onClick={onConfirmDeauthorization}
+              disabled={
+                !isDeauthorizationReqestActive ||
+                remainingAuthorizationDecreaseDelay !== "0"
+              }
+            >
+              Confirm Deauthorization
+            </Button>
+          </InfoBox>
+          <BodySm mt="4" color="gray.500">
+            <InfoIcon verticalAlign="sub" />{" "}
+            <Box as="span">
+              Increasing or decreasing the authorization amount is suspended
+              until the pending deauthorization is confirmed.
+            </Box>
+          </BodySm>
```

Comment:
Non-blocking:

This `InfoIcon` and `Box` doesn't seem to be aligned well:
<img width="193" alt="image" src="https://user-images.githubusercontent.com/40306834/193016847-f8b4fb75-f3a4-4861-9e99-f47f953a0e41.png">


I would propose to wrap it in `VStack` with middle align:

```suggestion
          <BodySm mt="4" color="gray.500">
            <HStack alignItems={"middle"}>
              <InfoIcon />{" "}
              <Box as="span">
                Increasing or decreasing the authorization amount is suspended
                until the pending deauthorization is confirmed.
              </Box>
            </HStack>
          </BodySm>
```

Result:
<img width="198" alt="image" src="https://user-images.githubusercontent.com/40306834/193016709-2ff7577e-782e-426b-a790-7a0174e8e651.png">

Reply:
5a629cd